src/components/draggable/draggable.test.js

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import DraggableContainer from "./draggable-container.component";
+import DraggableItem from "./draggable-item.component";
+import { Checkbox } from "../checkbox";
+
+import {
+  draggableItem,
+  draggableItemByPosition,
+} from "../../../cypress/locators/draggable";
+
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+const DraggableCustom = ({ getOrder, ...props }) => {
+  const handleUpdate = (items) => {
+    console.log(items);
+    if (getOrder) {
+      getOrder();
+    }
+  };
+
+  return (
+    <DraggableContainer {...props} getOrder={handleUpdate}>
+      <DraggableItem key="1" id={1}>
+        <Checkbox label="Draggable Label One" />
+      </DraggableItem>
+      <DraggableItem key="2" id={2}>
+        <Checkbox label="Draggable Label Two" />
+      </DraggableItem>
+      <DraggableItem key="3" id={3}>
+        <Checkbox label="Draggable Label Three" />
+      </DraggableItem>
+      <DraggableItem key="4" id={4}>
+        <Checkbox label="Draggable Label Four" />
+      </DraggableItem>
+    </DraggableContainer>
+  );
+};
+
+context("Test for Draggable component", () => {
+  describe("Check functionality for Draggable component", () => {
```

Comment:
describe("check functionality for Draggable component", () => {

Reply:
fixed